Makefile

```diff
@@ -0,0 +1,13 @@
+package_name = dokuwiki-plugin-icalevents
+version=`awk '/date/{print $$2}' plugin.info.txt`
+
+default: dist
+
+all:
+
+dist:
+	set -ex
```

Comment:
this is useless; each line in makefile is ran as in separate shell script. and `-e` behaviour is default in make and make will print out commands it runs anyway, so `-x` is also covered.

if you want to run in same shell script you should backslash escape the newlines:
```
set -xe; \
more stuff
```